cypress/integration/sortTableDom.spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+Cypress.Commands.add('isSorted', (col, number) => {
+  cy.contains(col).click();
+
+  cy.get(`tbody tr:nth-child(n) td:nth-child(${number})`).then(($fcolumn) => {
+    const columns = [...$fcolumn].map((column) => column.innerText);
+    const sortedColumns = [...columns].sort((a, b) => a - b);
```

Comment:
[...columns] -- why do you need this?

Reply:
it is a copy of the default array surely sorted. in the loop, we will compare each element of two arrays, if they are not equal then the column wasn't sorted.

Reply:
agree, I missed that.